docker/api/volume.py

```diff
@@ -0,0 +1,50 @@
+import functools
+
+from .. import errors
+from ..utils import utils
+
+
+def check_api_version(f):
```

Comment:
I'd suggest parameterizing this decorator with the version, so it can be used anywhere.